modules/mw.cx.TargetArticle.js

```
@@ -208,13 +214,46 @@ mw.cx.TargetArticle.prototype.publishSection = function () {
 mw.cx.TargetArticle.prototype.publishSuccess = function (response, jqXHR) {
 	const publishAction = this.translation.isSectionTranslation() ? 'cxpublishsection' : 'cxpublish';
 	const publishResult = response[publishAction];
+	console.log("publishResult:");
```

ESLint

[error] Unexpected console statement.

(no-console)